kit/inputs/src/useApplyMask/amount/runUIGetSeparatorRegExp.ts

```diff
@@ -0,0 +1,27 @@
+export function runUIGetSeparatorRegExp(
+    integerSeparator: string,
+    fractionalSeparator: string,
+    inputSeparators: string[],
+): RegExp {
+    'worklet';
+
+    const filteredSeparators = inputSeparators.reduce((result: string[], separator: string) => {
+        if (separator === integerSeparator || separator === fractionalSeparator) {
+            /** The currently used separators should not be used as delimeter */
+            return result;
+        }
+        // TODO Add more specific chars
+        if (separator === '.' || separator === '/') {
+            const separatorWithEscapeCharacter = `\\${separator}`;
+            result.push(separatorWithEscapeCharacter);
+            return result;
+        }
```

Comment:
This also scares me)

Reply:
We need to escape specific characters. I added full list of specific characters on top of the file.